### PR TITLE
User/ericjohnson327/transform fusion manifest

### PR DESCRIPTION
**Pull request by EricJohnson327**
Use xslt to create a fusion manifest from the application manifest.  This manifest will be put into a common 'manifests' folder to support unpackaged in-app use.

### CODE REVIEW

File: build/AzurePipelinesTemplates/WindowsAppSDK-CreateNugetPackage-Job.yml [hunk 1]
```diff
@@ -64,6 +64,11 @@ jobs:
         Copy-Item -Path "$targetsFilePath\Microsoft.WindowsAppSDK.Bootstrap.targets" -Destination "$fullpackagePath\build\Microsoft.WindowsAppSDK.Bootstrap.targets"
 
         Copy-Item -Path "$targetsFilePath\AppxManifest.xml" -Destination "$fullpackagePath\AppxManifest.xml"
+        $manifestPath = $fullpackagePath+'\manifests';
+        New-Item -ItemType Directory -Force -Path $manifestPath;
+        $xslt = New-Object System.Xml.Xsl.XslCompiledTransform;
+        $xslt.Load('build\TransformAppxManifest.xslt');
+        $xslt.Transform($fullpackagePath+'\AppxManifest.xml', $manifestPath+'\WindowsAppSDK.manifest');
```
Comment (Scottj1s) [inline build/AzurePipelinesTemplates/WindowsAppSDK-CreateNugetPackage-Job.yml:71]: I would name the output something like Microsoft.WindowsAppSdk.Foundation.manifest, just to ensure it doesn't collide with other transport packages, and to provide a pattern to follow

**Review by Scottj1s — APPROVED** [review]

File: build/AzurePipelinesTemplates/WindowsAppSDK-CreateNugetPackage-Job.yml (hunk 1 above)
Comment (EricJohnson327) [reply build/AzurePipelinesTemplates/WindowsAppSDK-CreateNugetPackage-Job.yml]: Good thinking.  Updated

Comment (DrusTheAxe) [reply build/AzurePipelinesTemplates/WindowsAppSDK-CreateNugetPackage-Job.yml]: @Scottj1s where should that be noted in the Transport Package Spec? ;-)

Comment (EricJohnson327) [reply build/AzurePipelinesTemplates/WindowsAppSDK-CreateNugetPackage-Job.yml]: @DrusTheAxe I'm planning to make spec updates once the end-to-end is working

**Pull request merged**